docs/app/pages/components/components-sections/facets/facet-check-list/facet-check-list.component.html

```diff
@@ -40,6 +40,9 @@
     <tr uxd-api-property name="simplified" type="boolean" defaultValue="false">
         Defines whether or not the checkboxes will appear in simplified form.
     </tr>
+    <tr uxd-api-property name="id" type="string">
+        This will allow you to add a unique id to the Facet Check List and allow that id to propagate down to its child components.
```

Comment:
I suggest using the same documentation format as the [number picker](https://pages.github.houston.softwaregrp.net/caf/ux-aspects-micro-focus/#/ui-components/input-controls#number-picker).